### ISSUE TITLE
Require Node.js 6?

### ISSUE THREAD

**Issue by realityking** [imagemin/meta#18]
I've been looking into updating several packages to reduce the amount of duplication in a typical imagemin installation, right now it's a bit all over the place.

I noticed some of the packages far down the tree are already requiring Node.js 6, e.g. [import-lazy](https://github.com/sindresorhus/import-lazy), [is-jpg](https://github.com/sindresorhus/is-jpg) and [semver-regex](https://github.com/sindresorhus/semver-regex).

**Should imagemin + plugins move to Node.js 6 as the min. required version?**

I'm happy to put the leg work in for this.

**Comment by sindresorhus** [comment]
Yes, we should drop Node.js 4 in all the modules. Help would be much appreciated.
